src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeInfo.cs

```diff
@@ -159,10 +163,13 @@ public static Task<SyntaxTreeDeclarationInfo> GetDeclarationInfoAsync(Document d
                             containsElementAccess = containsElementAccess || syntaxFacts.IsElementAccessExpression(node);
                             containsIndexerMemberCref = containsIndexerMemberCref || syntaxFacts.IsIndexerMemberCRef(node);
 
-                            DeclaredSymbolInfo declaredSymbolInfo;
-                            if (syntaxFacts.TryGetDeclaredSymbolInfo(node, out declaredSymbolInfo))
+                            if (!isGeneratedCode)
                             {
-                                declaredSymbolInfos.Add(declaredSymbolInfo);
+                                DeclaredSymbolInfo declaredSymbolInfo;
+                                if (syntaxFacts.TryGetDeclaredSymbolInfo(node, out declaredSymbolInfo))
+                                {
+                                    declaredSymbolInfos.Add(declaredSymbolInfo);
```

Comment:
Why are we cool with tokens in generated files? Can we just bail right at the top after realizing this is a generated file? If we really do just want to process tokens in generated files then an explanation would be useful.